DFC.ServiceTaxonomy.GraphSync/GraphSyncers/Helpers/EmbeddedContentItemsGraphSyncer.cs

```diff
@@ -127,6 +132,13 @@ public abstract class EmbeddedContentItemsGraphSyncer : IEmbeddedContentItemsGra
             }
         }
 
+        private IMergeGraphSyncer GetNewMergeGraphSyncer()
+        {
+            _logger.LogDebug("Getting new IMergeGraphSyncer.");
```

Comment:
Do you still need your debug log messages?

Reply:
They were added to help with making structural changes to the sync code, that because of recursion, callbacks and all the moving parts, makes it too cumbersome to step through it all.

You're right in that they're too chatty for logging as info (although some might be worth switching), but I think them worthwhile keeping to help with future work or bug fixes.